cluster/charts/crossplane-types/crds/database.crossplane.io_postgresqlinstances.yaml

```diff
@@ -146,12 +146,10 @@ spec:
               type: object
             engineVersion:
               description: 'EngineVersion specifies the desired PostgreSQL engine
-                version. Allowed Versions: 9.4, 10.0, 11.0 and 12.0.'
+                version. Allowed Versions: 9.6 and 11.'
               enum:
-              - "9.4"
-              - "10.0"
-              - "11.0"
-              - "12.0"
+              - "9.6"
```

Comment:
Did you run `make reviewable` after removing the validation line? I'd expect the `enum` array here to go away automatically since CRDs are deleted and regenerated via `make reviewable`

Reply:
No. But I did just now following with another commit. Thank you, now I learnt the mechanism.